### PR TITLE
feat(favorites): user favorites 

### PR DESCRIPTION
**Pull request by davidvader**

### CODE REVIEW

File: src/elm/Favorites.elm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+-- VIEW
+
+
+{-| starToggle : takes org repo msg and favorited status and renders the favorites toggle star
+-}
+starToggle : Org -> Repo -> ToggleFavorite msg -> Bool -> Html msg
+starToggle org repo toggleFavorite favorited =
+    star
```
Comment (wass3r) [inline src/elm/Favorites.elm]: this probably needs to get marked up better for accessibility

File: cypress/integration/favorites.spec.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2019 Target Brands, Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (davidvader) [reply cypress/integration/favorites.spec.js]: done

File: src/elm/Favorites.elm (hunk 1 above)
Comment (davidvader) [reply src/elm/Favorites.elm]: done

**Comment by davidvader** [comment]
adding a couple more accessibility features

**Pull request merged**